### DISCUSSION TITLE
Doodba development mode fails with command-line --dev=reload option

### DISCUSSION THREAD

**Discussion by BernatPForgeFlow** [Tecnativa/doodba#527] [General]
## Describe the bug
This is not happening to everyone, but I've seen this problem to some of my coworkers, included me. 

In development mode, Odoo doesn't respond to browser requests when --dev=reload option is set. The browser stays blank, and the logs keep saying the following: _socat[13] E getaddrinfo("odoo", "NULL", {1,0,1,6}, {}): Try again_

An example of the error that appears in the logs:
![image](https://user-images.githubusercontent.com/90243017/196446701-9e854dff-bd86-4182-8730-8f0865dd6b51.png)

Do you know what is happening here?

**Comment by pedrobaeza** [comment]
This is a side effect of the auto-save feature of some IDEs, like VSCode. Odoo is not able to correctly handle several reload requests, and crashes. The options are:

- Disable auto-save, and don't press Ctrl+S (or whatever save combination you have) very quickly.
- Disable reload option.
- Get Odoo to fix this, which is very difficult...

Passing this to a discussion.
